backend/middlewares/index.ts

```
@@ -26,9 +29,35 @@ const validateRequest = (schema: Schema): RequestHandler => {
 
 };
 
+/*
+ * Middleware to check if the user is an admin
+ */
+const adminAuth = (req: Request, res: Response, next: NextFunction): void => {
+	const token = req.headers.authorization?.split(' ')[1];
+
+	if (!token) {
+		return next(new Unauthorized('No token provided'));
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)